packages/opc-feedback/README.md

```diff
@@ -0,0 +1,185 @@
+# opc-feedback Component 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+Feedback Component is used to collect feedback from end user.
+
+## Install
+
+```sh
+npm install --save @one-platform/opc-feedback
+```
+## Themes
+| Theme Name   | Example |
+|---------|------------------------------------------------------------------|
+| red (*default and fallback color*) | ```<opc-feedback theme="red"></opc-feedback>``` |
+| blue | ```<opc-feedback theme="blue"></opc-feedback>``` |
+
+### Color pallet
```

Comment:
correction pallet spelling is wrong, it should be pallette

Reply:
corrected spelling